qfieldcloud_sdk/sdk.py

```diff
@@ -408,6 +410,16 @@ def upload_files(
 
         local_files = self.list_local_files(project_path, filter_glob)
 
+        # Validate file names
+        for file in local_files:
+            try:
+                is_valid_filename(file["name"], platform="auto")
+                is_valid_filepath(file["absolute_filename"], platform="auto")
+            except ValidationError as e:
+                raise ValueError(
+                    f"Invalid file name or path: {file['name']}. Error: {e}"
+                )
+
```

Comment:
```suggestion
        for file in local_files:
            is_valid_filepath(file["name"])
```

Leave it as simple as that, the error should be processed by the global error handler in the CLI or the SDK user.

Reply:
Also note that you seem to have confused the `file["name"]` to be the basename, which is not true, it is the relative file path of the file. We do not care about the `absolute_filename`, as it might contain directories that are not relevant for QFieldCloud, as they are only available locally on the machine of the uploading party.

Reply:
Actually do we even need this here, as we already do the validation in `upload_file`?

Reply:
You are right, is only required in the upload_file method!